framework-examples/aurelia/package.json

```diff
@@ -1,147 +1,137 @@
-  "contributors": [
-    {
-      "name": "Arjen de Blok"
+    "name": "aurelia2",
+    "version": "1.1.2",
+    "description": "A starter kit for building a standard navigation-style app with Aurelia and webpack.",
+    "main": "dist/main.js",
+    "scripts": {
+        "pretest": "del-cli \"test/coverage\"",
+        "test": "cross-env NODE_ENV=test TS_NODE_PROJECT=tsconfig.tsnode.json TS_NODE_FAST=true ts-node ./node_modules/karma/bin/karma start test/karma.conf.js",
+        "test:debug": "npm test -- --single-run=false --debug",
+        "webdriver:update": "cross-env ./node_modules/.bin/webdriver-manager update",
+        "webdriver:start": "cross-env ./node_modules/.bin/webdriver-manager start",
+        "pree2e": "npm run webdriver:update -- --standalone",
+        "e2e": "concurrently --success first --kill-others \"npm run e2e:start-when-ready\" \"cross-env WEBPACK_PORT=19876 npm start\"",
+        "e2e:start-when-ready": "wait-on --timeout 120000 http-get://localhost:19876/index.html && npm run e2e:start",
+        "e2e:start": "cross-env ./node_modules/.bin/protractor test/protractor.conf.js",
+        "e2e:live": "npm run e2e:start -- --elementExplorer",
+        "clean": "npm cache clean && del-cli node_modules \"test/coverage\" dist",
+        "clean:dist": "del-cli dist",
+        "preclean:install": "npm run clean",
+        "clean:install": "npm install",
+        "preclean:start": "npm run clean",
+        "clean:start": "npm start",
+        "watch": "npm run watch:dev",
+        "watch:dev": "npm run build:dev -- --watch",
+        "watch:dev:hmr": "npm run watch:dev -- --hot",
+        "watch:test": "npm run test -- --auto-watch --no-single-run",
+        "watch:prod": "npm run build:prod -- --watch",
+        "build": "npm run build:dev",
+        "prebuild:dev": "npm run clean:dist",
+        "build:dev": "cross-env NODE_ENV=development npm run webpack -- --progress --profile",
+        "prebuild:prod": "npm run clean:dist",
+        "build:prod": "cross-env NODE_ENV=production npm run webpack -- --progress --profile",
+        "start": "npm run server:dev",
+        "server": "npm run server:dev",
+        "server:dev": "cross-env NODE_ENV=development npm run webpack-dev-server -- --inline --progress --profile --watch",
+        "server:dev:hmr": "npm run server:dev -- --hot",
+        "server:prod": "http-server dist --cors",
+        "webpack": "cross-env TS_NODE_PROJECT=tsconfig.tsnode.json TS_NODE_FAST=true ./node_modules/.bin/webpack",
+        "webpack-dev-server": "cross-env TS_NODE_PROJECT=tsconfig.tsnode.json TS_NODE_FAST=true ./node_modules/.bin/webpack-dev-server"
     },
-    {
-      "name": "Bazyli Brzóska <bazyli.brzoska@gmail.com> (https://invent.life)"
-    }
-  ],
-  "bugs": {
-    "url": "https://github.com/aurelia/skeleton-navigation/issues"
-  },
-  "homepage": "https://github.com/aurelia/skeleton-navigation#readme",
-  "aurelia": {
-    "build": {
-      "resources": []
+    "repository": {
```

Comment:
Ah yeah I'd take that out as well. Pretty much anything that has to do with the Aurelia Skeleton, since it doesn't help convey the fact that this is an MDC-Web example.